source/MaterialXTest/Render.cpp

```diff
@@ -1119,7 +1125,11 @@ void printRunLog(const ShaderValidProfileTimes &profileTimes, const ShaderValidT
     {
         profilingLog << "---------------------------------------" << std::endl;
 
-        // Get implementation count from libraries
+        const std::string OSL_STRING("osl");
+        const std::string GEN_OSL_STRING(mx::OslShaderGenerator::LANGUAGE);
```

Comment:
Just wondering why you don't just use mx::OslShaderGenerator::LANGUAGE? Why bother copying the constant?

Reply:
Good point. I'll fix.